docs/_includes/navigation-dropdown.html

```diff
@@ -0,0 +1,20 @@
+<nav class="navbar navbar-toggleable-md navbar-light bg-faded">
+    <div class="container">
+        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#main-nav" aria-controls="main-nav" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
`.navbar-toggler-right` class isn't needed here; positioning of the toggler button is handled by its placement in the markup in Athena.  To right-align the button, add it after `.navbar-brand`.